query-engine/driver-adapters/js/connector-test-kit-executor/src/index.ts

```diff
@@ -132,15 +144,15 @@ async function handleRequest(method: string, params: unknown): Promise<unknown>
             const castParams = params as TeardownPayload;
             await schemas[castParams.schemaId].disconnect("")
             delete schemas[castParams.schemaId]
+            delete adapters[castParams.schemaId]
```

Comment:
Future refactoring: maybe turn `schemas`, `adapters` and `queryLogs` into a single
```ts
state: Record<
  number,
  {
    schema: engines.QueryEngineInstance,
    adapter: ErrorCapturingDriverAdapter,
    queryLogs: string[],
  }
>
```

so we don't have to manage each of the part separately and can't forget to delete any of them

Reply:
Absolutely.